mcc/odin/odin/snapshot_list.go

```diff
@@ -0,0 +1,18 @@
+package odin
+
+import (
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
+)
+
+// ListSnapshots return a list of all DBSnapshots.
+func ListSnapshots(
+	instanceName string,
+	svc rdsiface.RDSAPI,
+) (
+	result []*rds.DBSnapshot,
+	err error,
+) {
+	result = make([]*rds.DBSnapshot, 0)
+	return
```

Comment:
This is the implied return that @christofdamian loves from Go ;P
Since the returned values defined in the declaration are named, they are initialized to `nil`, and `return`, without any parameters, will take these named variables.